HOWTO.md

```diff
@@ -16,10 +16,11 @@
   - `serviceinfo_api_server.yml`
 - How to run the servers
   - Manufacturing Server
+  - Manufacturing Client
```

Comment:
The Manufacturing Client is a client, so we should move it to the `How to run the Client` section, you can add two different sections there for the `fdo-client-linuxapp` and the `fdo-manufacturing-client`